library/core/registry/src/main/java/org/quiltmc/qsl/registry/api/event/RegistryEntryPredicate.java

```diff
@@ -0,0 +1,13 @@
+package org.quiltmc.qsl.registry.api.event;
+
+import net.minecraft.util.Identifier;
+
+@FunctionalInterface
+public interface RegistryEntryPredicate<V> {
```

Comment:
Would be nice to have documentation in the module.

Reply:
Javadocs are to be completed, hence the PR being a draft